metrics/edfi_tech_metrics/ossf_score.py

```diff
@@ -13,7 +11,7 @@ def get_ossf_score(organization, repository):
     response = requests.get(url)
     if response.status_code == 200:
         data = response.json()
-        return data["score"]
+        return data['score']
```

Comment:
`"` is actually our preferred style. Not that it makes much difference. Main point is to be consistent.